app/res/xml/preferences_developer.xml

```diff
@@ -6,8 +6,8 @@
         android:key="cc-superuser-enabled"
         android:title="Developer Mode Enabled"/>
     <ListPreference
```

Comment:
Hm, I'm not seeing that in most of the prefs .xml files, but also honestly isn't a big deal so fine to just leave!